migration/v0.34/README.md

```diff
@@ -94,6 +94,22 @@ As with all scripts in this repository `CK8S_CONFIG_PATH` is expected to be set.
     git switch -d v0.34.x
     ```
 
+1. Verify that no important resources are present in the `elastic-system` and `influxdb-prometheus` namespaces.
+
+    ``` bash
+    ./bin/ck8s ops kubectl sc get all -n elastic-system
+    ./bin/ck8s ops kubectl sc get all -n influxdb-prometheus
```

Comment:
this (migration file)[https://github.com/elastisys/compliantkubernetes-apps/tree/d0d207a532078d64d084b1398b7b0401cad657d5/migration/v0.33#prepare-upgrade---non-disruptive] shows it better manual method is more like do each individual prepare/apply file instead of doing the whole prepare/apply file  that the automatic method does 
```suggestion
    ./migration/v0.34/prepare/30-delete-obsolete-ns.sh
```